### PR TITLE
Added Layout

### PR DESCRIPTION
**Pull request by deanvdRR**
Added Layout for all Features will wait for features to be uploaded to link to layout @jaderabbit

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -22,10 +22,80 @@
     <title>React App</title>
   </head>
   <body>
+  <style type="text/css">
```
Comment (jaderabbit) [inline public/index.html]: stylesheet pls

File: public/index.html [hunk 2]
```diff
@@ -22,10 +22,80 @@
...
+
+    <h1 id="header">Header</h1>
+    <div id="side">Side</div>
+    <div id="main">
+      <div class="content">
+
+      </div>
+
+    </div>
+  <!--<div id="floatingRectangle">Hello</div> -->
```
Comment (jaderabbit) [inline public/index.html]: removed unnecessary comments

File: src/index.js [hunk 3]
```diff
@@ -3,6 +3,16 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import registerServiceWorker from './registerServiceWorker';
+import ChatComponent from "./ChatComponent";
+import LoginComponent from "./LoginComponent";
+import HeaderComponent from "./HeaderComponent";
+
+
+ReactDOM.render(<LoginComponent />, document.getElementById('side'));
```
Comment (jaderabbit) [inline src/index.js]: Fix as per reinhardts explanation

**Review by jaderabbit — CHANGES_REQUESTED** [review]

**Comment by deanvdRR** [comment]
changed as recomended

**Pull request closed without merging**